### DISCUSSION TITLE
I want to write my blog only in English.

### DISCUSSION THREAD

**Discussion by burninghering** [MrGreensWorkshop/MrGreen-JekyllTheme#168] [How to] [answered]
![image](https://user-images.githubusercontent.com/37091602/216985059-c288997c-3f1c-44e7-a2a4-08e8c62eefdf.png)

I want to remove the marked place to display the blog only in English. Is there a way?

**Comment by MrGreensWorkshop** [comment, marked as answer]
Check out [Mr. Green theme tutorials playlist](https://www.youtube.com/playlist?list=PLAymxPbYHgl-fFy5can7uZBMJtFWVcphD) on YouTube.
